gef-ui/server/controllers/amendments/eligibility-criteria/get-eligibility.ts

```diff
@@ -41,12 +41,13 @@ export const getEligibility = async (req: GetEligibilityRequest, res: Response)
       return res.redirect('/not-found');
     }
 
-    // TODO 7765: Fetch criteria using GET endpoint from database and add to viewModel
+    const { criteria } = amendment.eligibilityCriteria;
```

Comment:
```suggestion
    const { criteria } = amendment?.eligibilityCriteria;
```

Reply:
This shouldn't be needed as `eligibilityCriteria` is a required field on the amendment. But I've added in the ? anyway

Reply:
Actually adding in the `?` causes problems elsewhere in the file. This isn't needed as it is a required field on amendments (and when we fetch the amendments we have `safeParse` validation to ensure it exists). So leaving as is